Automattic/jetpack
# Social Icon Widget: Fix Icons Not Saving

Fixes #20698

#### Changes proposed in this Pull Request:

* Uses the suggestion provided by @kbrown9 in https://github.com/Automattic/jetpack/issues/20698#issuecomment-907215640 to fix issue where widget doesn't save in the wp-admin widget editor.
#### Jetpack product discussion


https://github.com/Automattic/jetpack/issues/20698#issuecomment-907215640
#### Does this pull request change what data or activity we track or use?


No
#### Testing instructions:






* Enable Jetpack widgets in the Jetpack -> Writing settings
* Go to Appearance -> Widgets and add the Jetpack Social Icon widget.
* Add a few icons. 
* Make sure they show in the preview. 
* Click update. The icons should stay. Make sure they work on the front end.
* Go to Appearance -> Customizer and make sure the Jetpack Social Icons also works in the customizer following the same steps above.
* There should be no related php errors. 

This PR does not fix the issue where removing an icon doesn't let you update the widgets; that is being tracked over at https://github.com/Automattic/jetpack/issues/20846
